freeze.py

```diff
@@ -175,7 +175,7 @@ def find_files(directory, patterns):
             git_log_filepath = os.path.join(share_path, git_log_filename)
             if os.path.isfile(git_log_filepath):
                 src_files.append((git_log_filepath, "settings/%s" % git_log_filename))
-                if os.path.splitext(git_log_filepath)[1] == "":
+                if os.path.splitext(git_log_filepath)[1] == ".env":
                     # No extension, parse version info
```

Comment:
Heh. At the risk of beating a dead horse, that change made this comment a lie. :wink: